ledger-core/virtual/integration/method_test.go

```diff
@@ -137,6 +168,37 @@ func TestVirtual_Method_WithoutExecutor(t *testing.T) {
 	}
 }
 
+func TestVirtual_Method_WithExecutor_ObjectIsNotExist(t *testing.T) {
+	t.Log("C4974")
+	t.Skip("https://insolar.atlassian.net/browse/PLAT-395")
+
+	server, ctx := utils.NewServer(nil, t)
+	defer server.Stop()
+
+	objectRef := reference.NewSelf(server.RandomLocalWithPulse())
+
+	msg := makeVStateReportWithState(server.GetPulse().PulseNumber, objectRef, payload.Missing, nil)
+	server.SendMessage(ctx, msg)
```

Comment:
I think you need to add `server.WaitIdleConveyor()` staff.

Reply:
server.WaitActiveThenIdleConveyor()